src/lib/analytics.js

```diff
@@ -67,14 +67,14 @@ module.exports = function() {
   }
 
   // 2 DP
-  function toTwoDP (i) {
+  function toTwoDP(i) {
```

Comment:
Rest of this file is just lint fixes. Not sure why it wasn't being caught before, but now it is.